src/main/kotlin/no/nav/syfo/dialogmote/database/domain/PMotedeltakerArbeidstakerVarsel.kt

```diff
@@ -33,4 +35,6 @@ fun PMotedeltakerArbeidstakerVarsel.toDialogmotedeltakerArbeidstaker() =
         lestDatoArbeidstaker = this.lestDato,
         fritekst = this.fritekst,
         document = this.document,
+        brevBestillingId = brevBestillingId,
```

Comment:
Burde det være this. foran her også? (Gjelder også i PReferat.toReferat)

Reply:
Er ikke sikkert det endrer semantikken, men det er klarere i alle fall.